azkaban-common/src/test/java/azkaban/utils/EmailerTest.java

```diff
@@ -1,56 +1,75 @@
+/*
+ * Copyright 2017 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
 package azkaban.utils;
 
+import com.google.common.io.Resources;
+import org.junit.Before;
 import org.junit.Test;
 
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.net.URL;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Properties;
+
 
-/**
- * Created by wqs on 17/1/11.
- */
 public class EmailerTest {
 
+    List<String> senderList = new ArrayList<String>();
+
+
+    @Before
+    public void setUp() throws Exception {
+        senderList.add("sender@domain.com");
+    }
 
     /**
      * test emailMessage properties
      */
     @Test
-    public void testCreateEmailMessage(){
-        List<String> senderList = new ArrayList<String>();
-        senderList.add("sender@xxx.com");
-        EmailMessage emailMessage = null;
+    public void testCreateEmailMessage() throws Exception{
+
+        URL url = Resources.getResource("conf/emailtest/azkaban.properties");
+        InputStream inputStream = new FileInputStream(url.getFile());
 
+        Properties properties = new Properties();
+        properties.load(inputStream);
```

Comment:
What's the advantage of loading the properties from the resource file instead of your previous approach?